sos/report/plugins/hbn.py

```diff
@@ -12,13 +12,28 @@
 
 class HBN(Plugin, IndependentPlugin):
 
-    short_desc = 'HBN (Host Based Network)'
+    short_desc = 'HBN (Host Based Networking)'
     plugin_name = "hbn"
+    vsctl = "ovs-vsctl"
     packages = ('hbn-repo',)
 
     def setup(self):
         self.add_copy_spec([
             '/etc/mellanox',
             '/var/log/doca/hbn',
             '/var/lib/hbn/etc/cumulus',
+            '/var/log/containers',
```

Comment:
there is a container plugin which will run based on the frontend (crictl, docker etc)

this is not needed.

Reply:
ok, will drop